extensions/amp-analytics/0.1/events.js

```diff
@@ -348,15 +354,14 @@ export class VisibilityTracker extends EventTracker {
   add(context, eventType, config, listener) {
     const visibilitySpec = config['visibilitySpec'] || {};
     const selector = config['selector'] || visibilitySpec['selector'];
+    const waitForSpec = visibilitySpec['waitFor'];
```

Comment:
Let's update the docs right away. Search for "The element visibility trigger waits for element's ini-load signal before matching the visibilitySpec."